src/code_translation/input/pandas.py

```diff
@@ -27,6 +27,14 @@ def visit_df_merge(self, df_node: DataFrameNode, *args, **kwargs):
     def visit_df_set_index(self, df_node: DataFrameNode, *args, **kwargs):
         return SetKeyNode(df_node, *args, **kwargs)
 
+    def visit_df_aggregate(self, df_node: DataFrameNode, func: str, *args, **kwargs):
+        # func could be '"max"'
+        func = func.replace('"', "")
```

Comment:
could we please consider making the resolve_SimpleString method trim these automatically (preferrably by removing the first and last character instead of every '"')

Reply:
We can think of a solution that scales better.